server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
Ok, the link you posted has exported values with the name Metadata.

Pertaining directly to this question is this:

https://github.com/golang/tools/blob/5a2fc32f4475381b7c41823909c5ec098f07c040/godoc/meta.go
40 // If no metadata is present the original byte slice is returned.
41	//
42	func extractMetadata(b []byte) (meta Metadata, tail []byte, err error) {
43		tail = b
44		if !bytes.HasPrefix(b, jsonStart) {
…	
52		if err = json.Unmarshal(b, &meta); err != nil {
53			return
54		}
55		tail = tail[end+len(jsonEnd):]
56		return
57	}

Exported variable? Yes, Metadata. Unexported value? concise weighs out pedantic spelling things out: meta.